client/lib/form-state/index.js

```diff
@@ -242,12 +241,16 @@ function setFieldErrors( formState, fieldErrors, hideFieldErrorsOnChange ) {
 	);
 }
 
+function getDefaultFieldState() {
+	return {
+		isShowingErrors: true,
+	};
+}
+
 function showAllErrors( formState ) {
 	return updateFields(
 		initializeFields( formState, getAllFieldValues( formState ) ),
-		constant( {
-			isShowingErrors: true,
-		} )
+		getDefaultFieldState
```

Comment:
I'd personally write the callback inline here. Other usages of `updateFields` in the same file do that, too.

By the way, all the `updateFields` callback would benefit from becoming arrow functions.

Reply:
> I'd personally write the callback inline here. Other usages of updateFields in the same file do that, too.

Sure, I've moved it inline.

> By the way, all the updateFields callback would benefit from becoming arrow functions.

I'm consciously refraining from unrelated changes in general.